src/Transport/Creation/AzureServiceBusForwardingSubscriptionCreator.cs

```diff
@@ -2,12 +2,13 @@
 {
     using System;
     using System.Collections.Concurrent;
+    using System.Linq;
     using System.Threading.Tasks;
     using Microsoft.ServiceBus.Messaging;
     using Logging;
     using Settings;
 
-    class AzureServiceBusForwardingSubscriptionCreator : ICreateAzureServiceBusSubscriptions
+    class AzureServiceBusForwardingSubscriptionCreator : ICreateAzureServiceBusSubscriptions, ICreateAzureServiceBusSubscriptionsAbleToDeleteSubscriptions
```

Comment:
This interface shouldn't be public, right? https://github.com/Particular/NServiceBus.AzureServiceBus/blob/develop/src/Transport/Creation/ICreateAzureServiceBusSubscriptions.cs#L12